types/ckeditor__ckeditor5-alignment/index.d.ts

```diff
@@ -1,8 +1,16 @@
-// Type definitions for @ckeditor/ckeditor5-alignment 27.0
+// Type definitions for @ckeditor/ckeditor5-alignment 27.1
 // Project: https://ckeditor.com/docs/ckeditor5/latest/api/alignment.html
 // Definitions by: Federico Panico <https://github.com/fedemp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.2
-export { default as Alignment } from "./src/alignment";
-export { default as AlignmentEditing } from "./src/alignmentediting";
-export { default as AlignmentUI } from "./src/alignmentui";
+import Alignment from './src/alignment';
+import AlignmentEditing from './src/alignmentediting';
+import AlignmentUI from './src/alignmentui';
+
+declare const _default: {
```

Comment:
```ts
export {
	Alignment,
	AlignmentEditing,
	AlignmentUI,
}
```
there is no defualt export IMO:
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-alignment/src/index.js